aes/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+project (aes C)
+
+set (SRC host/main.c)
+
+add_executable (${PROJECT_NAME} ${SRC})
+
+include_directories(${OPTEE_EXAMPLES_SDK}/host_include)
```

Comment:
In the future it would be nice to use something like `target_include_directories(ta-dev-kit PUBLIC include)`, but I believe as it is right now it will be hard to achieve that, so this will do.

Reply:
I didn't pay attention first, this is about host only and not TA's as stated in the commit message. So is this even needed? I can't see any files being included from `host_include` 
```bash
$ find . -name "*.[ch]" | xargs grep "#include" | grep host
./hotp/host/main.c:#include <err.h>
./hotp/host/main.c:#include <stdio.h>
./hotp/host/main.c:#include <string.h>
./hotp/host/main.c:#include <tee_client_api.h>
./hotp/host/main.c:#include <hotp_ta.h>
./hello_world/host/main.c:#include <err.h>
./hello_world/host/main.c:#include <stdio.h>
./hello_world/host/main.c:#include <string.h>
./hello_world/host/main.c:#include <tee_client_api.h>
./hello_world/host/main.c:#include <hello_world_ta.h>
./aes/host/main.c:#include <err.h>
./aes/host/main.c:#include <stdio.h>
./aes/host/main.c:#include <string.h>
./aes/host/main.c:#include <tee_client_api.h>
./aes/host/main.c:#include <aes_ta.h>
./random/host/main.c:#include <err.h>
./random/host/main.c:#include <stdio.h>
./random/host/main.c:#include <string.h>
./random/host/main.c:#include <tee_client_api.h>
./random/host/main.c:#include <random_ta.h>
```
You're after `tee_client_api.h` only? And that is in optee_client. This means that the dependencies are indeed being built with CMake with the introduction of CMake in optee_client. So, I think it should be possible to change this to something similar to what you have been doing with the `target_link_libraries` below. For example: `target_include_directories(teec PUBLIC include)`. What do you think?

Reply:
Thanks, that did the trick.